README.md

```diff
@@ -15,13 +15,17 @@ git clone https://github.com/Data4Democracy/collect-social.git
 Then install the requirements using `pip`. 
 
 ```bash
+cd collect-social
 pip install -r requirements.txt
+pip install .
```

Comment:
Not sure we need this step `pip install .` did you have issues when you tried without including this step?